.github/workflows/update-gov-zone-file.yml

```diff
@@ -0,0 +1,35 @@
+name: Upload gov.txt.gz from CZDS API
+run-name: Upload gov.txt.gz from CZDS API
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: "5 6 * * *"
+  push:
+    branches:
+      - es/107-zone-file-script
+    paths:
+      - .github/workflows/update-gov-zone-file.yml
+
+jobs:
+  update_zone_file:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download gov.txt.gz
+        env:
+          ICANN_USERNAME: ${{ secrets.ICANN_USERNAME }}
+          ICANN_PASSWORD: ${{ secrets.ICANN_PASSWORD }}
+        run: |
+          chmod +x ./scripts/download_zone_file.sh
+          ./scripts/download_zone_file.sh ${{ env.ICANN_USERNAME }} ${{ env.ICANN_PASSWORD }} > gov.txt
+        shell: sh
+      # Push daily zone file to branch only if a diff is found
+      - name: Commit and push updated zone file update to branch
+        run: |-
+          git config user.name "Continuous Integration - Zone File"
```

Comment:
Can we call our user `botgov`? 😄

Reply:
I like that a lot and laughed - will change it :D